### PR TITLE
node 0.10 compatibility

### PR DESCRIPTION
**Pull request by zoltan-mihalyi**
The task runs in node 0.10 after replacing const expressions to var.

### CODE REVIEW

**Comment by paazmaya** [comment]
Could you add it also in Travis list of versions?

File: package.json [hunk 1]
```diff
@@ -45,7 +45,7 @@
   },
   "devDependencies": {
     "grunt": "^1.0.1",
-    "grunt-eslint": "^19.0.0"
+    "grunt-eslint": "^18.0.0"
```
Comment (paazmaya) [inline package.json]: Why going back one version?

Comment (zoltan-mihalyi) [inline package.json]: The eslint task is not ES5 compatible in the 19.0 and it does not work in node 0.10.
If the eslint task could be skipped when running on older node versions this wouldn't be a problem, but I think this is the easiest solution.

Comment (zoltan-mihalyi) [inline package.json]: This commit is the problem:
https://github.com/sindresorhus/grunt-eslint/commit/d0ff94266c668a5d6ad2bfd5bb0e92445f6a580a

**Comment by paazmaya** [comment]
Thank you for your effort!

**Comment by zoltan-mihalyi** [comment]
your welcome!

**Pull request merged**